libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1785,6 +1785,7 @@ void ExpressionCompiler::appendVariable(VariableDeclaration const& _variable, Ex
 		setLValueFromDeclaration(_variable, _expression);
 	else
 	{
+		solUnimplementedAssert(_variable.value(), "Not implemented for constant parameters.");
```

Comment:
I was just copying the style I saw 10 lines above since @chriseth mentioned this was not implemented.